File: 20242Q/poo/laboratorio/tp04/ej3/src/MovablePoint.java
public class MovablePoint extends Point implements Movable{

    public MovablePoint(double x, double y) {
        super(x, y);
    }

    public void moveNorth(double delta) {
        y += y;
    }

    public void moveSouth(double delta) {
        y -= y;
    }

    public void moveEast(double delta) {
        x += x;
    }

    public void moveWest(double delta) {
        x -= x;
    }
}

File: 20242Q/poo/laboratorio/tp10/_soluciones/ej2/movable_point.rb
require_relative 'point'
require_relative 'movable'

class MovablePoint < Point
  include Movable

  def move_up(delta)
    @y += delta
  end

  def move_down(delta)
    @y -= delta
  end

  def move_left(delta)
    @x -= delta
  end

  def move_right(delta)
    @x += delta
  end
end
